NashvilleHistory/www/js/controllers/ToursCtrl.js

```diff
@@ -1,5 +1,89 @@
 'use strict';
 
-app.controller('ToursCtrl', function($scope, $state) {
+app.controller('ToursCtrl', function($scope, $state, $rootScope, CustomTourFact) {
+
+    // Listen for changes in State
+    $rootScope.$on('$stateChangeSuccess', function(event, toState){
+        if ($state.current.name == "app.editMyTour") {
```

Comment:
one way to update markers based on site navigation, need to research the ionic way